app/frontend/components/common/footer.js

```diff
@@ -23,19 +36,74 @@ const Footer = () =>
           'View on Github'
         )
       ),
-      '/',
+      ' /',
```

Comment:
this space is probably redundant since other links do not have this space before the slash